packages/hierarchies-react/src/presentation-hierarchies-react/itwinui/TreeRenderer.tsx

```diff
@@ -51,6 +63,9 @@ export function TreeRenderer({ rootNodes, expandNode, localizedStrings, selectNo
       <Tree.Root style={{ height: "100%", width: "100%" }}>
         <TreeLevelRenderer
           {...treeProps}
+          actions={[...(actions || []), getFilterAction({ onClick: onFilterClick, getHierarchyLevelDetails, label: localizedStrings?.filterHierarchyLevel })]}
```

Comment:
I think we should leave for consumer to add `filterButton`. That way it will be possible to fully control buttons position. I think we should have something like:

```tsx
// in consumer component
return <TreeRenderer
  {...treeProps}
  actions=[
    (node) => createMyCustomAction(node),
    createFilterAction({ 
      onFilter: handleFilter // same as `onFilerClick` at the moment
      getHierarchyLevelDetails
    })
  ]
/>
```

Reply:
Changed, the overflow into dropdown is not perfect and there could be cases where the overflow button will have one action, but I don't think there is any reason to add any more effort when itwinUI will change action logic. For filterButton exposed it with alpha tag, made label required for now consumers should have localizedStrings anyway should circle back to it when final version of itwinUI actions is available.